extensions/wikia/AdEngine/js/recovery/message.js

```diff
@@ -14,14 +20,54 @@ define('ext.wikia.adEngine.recovery.message', [
 
 	var logGroup = 'ext.wikia.adEngine.recovery.message',
 		wikiaTopAdsId = 'WikiaTopAds',
-		wikiaRailId = 'WikiaRail';
+		wikiaRailId = 'WikiaRail',
+		localstorageKey = 'rejectedRecoveredMessage',
+		headerText = 'Hey! It looks like you\'re using ad blocking software!',
+		messageText = 'Wikia runs ads so we can keep the lights on and so Wikia will always be free to use. ' +
+			'We can bring you fun, free, fan-oriented content until my glorious return to Earh if you ' +
+			'<strong>add us to your adblock whitelist</strong>. ' +
+			'<a class="action-accept">Click</a> my face to refresh after you\'re done!';
+
+	function accept() {
+		adTracker.track('recovery/message', 'accept');
+		location.reload();
+	}
+
+	function reject(message) {
```

Comment:
What do you think of changing `message` to `messageContainer` or `msgContainer`? I thinking about such semantic change because we're passing and hiding the element with the message not only the message ;)